README.md

```diff
@@ -28,38 +28,32 @@ This project is nascent and still under heavy development, but first steps:
 
 1.  Clone the repository:
 
-    ```bash
+    ```shell
     git clone https://github.com/tock/libtock-rs
     cd libtock-rs
     ```
 
-1.  Install `elf2tab`:
+1.  Install the dependencies:
 
-    ```bash
-    cargo install -f elf2tab --version 0.4.0
+    ```shell
+    make setup
```

Comment:
I wonder how you changed your mind here?
See: https://github.com/tock/libtock-rs/pull/157#discussion_r384128707

Reply:
Which bit? I haven't changed my mind, I just moved this from multiple commands to a single command.